### PR TITLE
Improvements v1

### PR DESCRIPTION
**Pull request by f3l1x**
# OLD 
selection_006
selection_005

# NEW 
selection_004
selection_003

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Thanks for PR! Very nice improvements.

On offline meetup 1 we agreed to push people directly to Slack, rather then create standalone box for every framework/CMS ther is.

As for meetups, outsource to Eventigo.

Ref #94

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/.idea
```
Comment (TomasVotruba) [inline .gitignore]: Btw, this should be in global `.gitignore` no to pollute repository by all IDEs:
 http://stackoverflow.com/a/7335487/1348344

File: app/config/sculpin_site.yml [hunk 2]
```diff
@@ -1,2 +1,3 @@
 title: "Děláš v PHP? Jsi jedním z nás"
 google_analytics_tracking_id: "UA-46082345-1"
+build: "v16112016"
```
Comment (TomasVotruba) [inline app/config/sculpin_site.yml]: What is this for?

File: source/_layouts/default.twig [hunk 3]
```diff
@@ -1,30 +1,31 @@
...
 <html lang="cs">
     <head>
-        <title>{% if page.title is empty %}{{ site.title }}{% else %}{{ page.title }}{% endif %}</title>
-
         <meta charset="utf-8">
-        <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+        <link href="/favicon.ico" rel="shortcut icon">
```
Comment (TomasVotruba) [inline source/_layouts/default.twig]: I see you've added the `favicon.ico` file. But it has over 360 kb. 

Could you make it smaller (~ 10 kb) and re-resend in standalone PR please?

File: source/potkavame-se.twig [hunk 4]
```diff
@@ -57,32 +41,51 @@ layout: default
...
+    </div>
+</div>
 
-                Cheš založit ve vlastním městě, napiš tomas
-                Chceš se přidat, napiš organizátorovi :)
+<div class="section intermezzo">
+    <div class="container">
+        <div class="row">
+            <div class="col-md-12 text-center">
+                Chceš uskutečnit sraz ve vlastním městě, napiš nám na
```
Comment (TomasVotruba) [inline source/potkavame-se.twig]: Nice call to action! Will add

**Comment by TomasVotruba** [comment]
:+1: for screenshot comparison, much easier to read!

**Comment by TomasVotruba** [comment]
Succeeded by #114

**Comment by TomasVotruba** [comment]
@f3l1x send those little improvements in new PR please

**Comment by f3l1x** [comment]
@TomasVotruba Sorry, I don't have much time at this moment. I thought that PR was OK, wasn't?

**Comment by TomasVotruba** [comment]
@f3l1x Ok.  I Will handle it. 
Check comments for details.

**Pull request closed without merging**